mainnet/pregenesis.json

```diff
@@ -0,0 +1,154 @@
+          "duration_until_decay": "15780000s",
+          "duration_of_decay": "10520000s",
+          "claim_denom": "ustarx"
+        }
+      },
+      "crisis": {
+        "constant_fee": {
+          "denom": "ustarx",
+          "amount": "500000000000"
+        }
+      },
+      "distribution": {
+        "params": {
+          "community_tax": "0.050000000000000000",
+          "base_proposer_reward": "0.010000000000000000",
+          "bonus_proposer_reward": "0.040000000000000000",
+          "withdraw_addr_enabled": true
+        }
+      },
+      "gov": {
+        "starting_proposal_id": "1",
+        "deposits": [],
+        "votes": [],
+        "proposals": [],
+        "deposit_params": {
+          "min_deposit": [
+            {
+              "denom": "ustarx",
+              "amount": "10000000000"
+            }
+          ],
+          "max_deposit_period": "1209600s"
+        },
+        "voting_params": {
+          "voting_period": "259200s"
+        },
+        "tally_params": {
+          "quorum": "0.200000000000000000",
+          "threshold": "0.500000000000000000",
+          "veto_threshold": "0.334000000000000000"
+        }
+      },
+      "ibc": {
+        "client_genesis": {
+          "params": {
+            "allowed_clients": ["07-tendermint"]
+          },
+          "create_localhost": false,
+          "next_client_sequence": "0"
+        }
+      },
+      "mint": {
+        "minter": {
+          "inflation": "0.200000000000000000",
+          "annual_provisions": "0.000000000000000000"
+        },
+        "params": {
+          "mint_denom": "ustarx",
+          "inflation_rate_change": "1.000000000000000000",
+          "inflation_max": "0.200000000000000000",
+          "inflation_min": "0.070000000000000000",
+          "goal_bonded": "0.670000000000000000",
+          "blocks_per_year": "4360000"
+        }
+      },
+      "slashing": {
+        "params": {
+          "signed_blocks_window": "10000",
+          "min_signed_per_window": "0.050000000000000000",
+          "downtime_jail_duration": "600s",
+          "slash_fraction_double_sign": "0.050000000000000000",
+          "slash_fraction_downtime": "0.000100000000000000"
+        }
+      },
+      "staking": {
+        "params": {
+          "unbonding_time": "1209600s",
+          "max_validators": 100,
+          "max_entries": 7,
+          "historical_entries": 10000,
+          "bond_denom": "ustarx"
+        }
+      },
+      "transfer": {
+        "port_id": "transfer",
+        "denom_traces": [],
+        "params": {
+          "send_enabled": true,
+          "receive_enabled": true
+        }
+      },
+      "upgrade": {},
+      "vesting": {},
+      "wasm": {
+        "params": {
+          "code_upload_access": {
+            "permission": "Nobody",
+            "address": ""
+          },
+          "instantiate_default_permission": "Everybody",
+          "max_wasm_code_size": "614400"
```

Comment:
Bump?

Reply:
Probably.. lemme compile a few contracts to get an idea of size.

Reply:
My thinking is that if contracts have to be approved, we might as well give ourselves leg room.

Reply:
Otherwise, some poor soul will have to submit two proposals to add their contract.

Reply:
A pretty large contract (cw20-staking) is 389Kb, so I think we're good here. We want to encourage small composable contracts anyway.